lib/CL/devices/spirv_parser.cc

```diff
@@ -1072,4 +1071,15 @@ bool applyAtomicCmpXchgWorkaround(const int32_t *InStream, size_t NumWords,
   return true;
 }
 
+bool applyAtomicCmpXchgWorkaroundInPlace(int32_t *InStream, size_t *NumWords) {
```

Comment:
For MSVC, the export attribute (`SPIRV_PARSER_EXPORT`) needs to be applied on the definition too.